pom.xml

```diff
@@ -74,6 +74,12 @@
         <rpkgtests-maven-plugin.version>0.8.0</rpkgtests-maven-plugin.version>
         <groovy-maven-plugin.version>2.1.1</groovy-maven-plugin.version>
         <useReleaseProfile>true</useReleaseProfile>
+
+        <!-- To run native tests, use
+
+             mvn clean verify -Dinvoker.goals="clean verify -e -Dnative"
+        -->
+        <invoker.goals>clean verify -e</invoker.goals>
```

Comment:
I think I can define `native` profile that will override the property. Then simply `-Pnative` will be enough.
Have to you tried cleaning your repo from all the universe artifact including the rpkgtests artifacts and running `clean verify`?

Reply:
> I think I can define `native` profile that will override the property.

+1, but please still leave `invoker.goals` overridable via CLI

Reply:
`verify` works. I remember trying it before and got resolution failures. I guess it was something else.